packages/api/README.md

```diff
@@ -64,16 +47,14 @@ We suggest passing provider as a string url, sdk will chose Provider Class based
 
 All `api.tx.<section>.<method>(...)` return CennznetExtrinsic, which have
 
-To use doughnut or feeExchange, it is required to pass them as part of SignerOption
+To use feeExchange, it is required to pass them as part of SignerOption
```

Comment:
could be clearer: 'To set fee exchange options on a transaction, it should be included as an additional argument when signing e.g:'